pod/main/static/js/main.js

```diff
@@ -305,8 +305,8 @@ var get_list = function (
   var prefix = "";
   for (i = 0; i < level; i++) prefix += "&nbsp;&nbsp;";
   if (level != 0) prefix += "|-";
-
-  tab.forEach((val) => {
+  for(var i = 0; i < tab.length; i++) {
```

Comment:
En effectuaunt des tests, je suis tombé sur l'erreur suivante :
`TypeError: undefined is not an object (evaluating 'tab.length')`
Il faudrait peut-etre mettre ca dans un "if (tab)", ou s'assurer que tab existe toujours ?

Reply:
je vais regarder, quels étaient les tests que tu faisais ?

Reply:
je regarde

Reply:
je viens de revoir ce fichier sur la partie theme en ajoutant des vérifications et j'ai du reprendre la partie ecoute d'evts. si on retirait des chaines du selecteur, elles n'étaient pas retirées de la liste des chaines...

Reply:
hum étrange de nouveau, je regarde ca de suite pour eviter de repaser en WIP car je de nouvelles modifs à faire, je vais rester dans cette branche

Reply:
je viens de proposer une nouvelle version avec un observer qui va attendre que le select2 soit pret avant de rajouter l'observer sur les channels
Merci de verifier